reference/docs-conceptual/learn/Writing-Portable-Modules.md

```diff
@@ -0,0 +1,301 @@
+---
+ms.date:  12/14/2018
+keywords:  powershell,cmdlet
+title:  Writing Portable Modules
+---
+
+# Portable Modules
+
+Portable modules are modules that work in both Windows PowerShell and PowerShell
+Core.
+Windows PowerShell is written for [.NET Framework](/dotnet/framework/)
+while PowerShell Core is written for [.NET Core](/dotnet/core/).
+
+While .NET Framework and .NET Core are highly compatible, there are differences
+in the available APIs between the two so modules intended to be used in both
+environments need to be aware of these differences.
+
+## Porting an Existing Module
+
+### Porting a PSSnapIn
+
+PowerShell SnapIns (PSSnapIn) are not supported in PowerShell Core.
+However, it is nearly trivial to convert a PSSnapIn to a PowerShell module.
+Typically, the PSSnapIn registration code is in a single source file of a
+class that derives from [PSSnapIn](/dotnet/api/system.management.automation.pssnapin?view=powershellsdk-1.1.0).
+Simply remove this source file from the build as it is no longer needed.
+
+Use `New-ModuleManifest`(/powershell/module/microsoft.powershell.core/new-modulemanifest?view=powershell-6)
+to create a new module manifest that replaces the need for the PSSnapIn
+registration code.  Some of the values from the PSSnapIn (such as Description)
+can be reused within the module manifest.
```

Comment:
Should you mention that the `RootModule` should be set to the dll of your portable module? Or speak to the mandatory items in the module manifest?

Reply:
Will add